docs_user/modules/proc_configuring-ldap-with-domain-specific-drivers.adoc

```diff
@@ -0,0 +1,99 @@
+user_objectclass = <user_object_class>
+user_id_attribute = <user_id_attribute>
+user_name_attribute = <user_name_attribute>
+user_mail_attribute = <user_mail_attribute>
+user_enabled_attribute = <user_enabled_attribute>
+user_enabled_default = true
+# Group configuration
+group_tree_dn = <group_tree_dn>
+group_objectclass = <group_object_class>
+group_id_attribute = <group_id_attribute>
+group_name_attribute = <group_name_attribute>
+group_member_attribute = <group_member_attribute>
+group_members_are_ids = true
+----
++
+* Replace the values, such as `<ldap_server_host>`, `<bind_dn_user>`, `<user_tree_dn>`, and so on, with your LDAP server details.
+
+. Create the secret from this file:
++
+----
+$ oc create secret generic keystone-domains \
+  -n <namespace> \
+  --from-file=<keystone.DOMAIN_NAME.conf>
+----
++
+* Replace `<namespace>` with the namespace where your {identity_service} is deployed.
+* Replace `<keystone.DOMAIN_NAME.conf>` with the name of your local configuration file. If applicable, include additional configuration files by using the `--from-file` option. After creating the secret, you can remove the local configuration file if it is no longer needed, or store it securely.
++
+[IMPORTANT]
+The name of the file that you provide to `--from-file`, for example `keystone.DOMAIN_NAME.conf`, is critical. The {identity_service} uses this filename to map incoming authentication requests for a domain to the correct LDAP configuration. Ensure that `DOMAIN_NAME` matches the name of the domain you are configuring in the {identity_service}.
+
+. Patch the `OpenStackControlPlane` CR:
++
+----
+$ oc patch openstackcontrolplane <cr_name> \
+  -n <namespace> \
+  --type=merge \
+  -p '
+spec:
+  keystone:
+    template:
+      customServiceConfig: |
+          [identity]
+          domain_specific_drivers_enabled = true
+      extraMounts:
+        - name: v1
+          region: r1
+          extraVol:
+            - propagation:
+              - Keystone
+              extraVolType: Conf
+              volumes:
+              - name: keystone-domains
+                secret:
+                  secretName: keystone-domains
+              mounts:
+              - name: keystone-domains
+                mountPath: "/etc/keystone/domains"
+                readOnly: true
+----
++
+* Replace `<cr_name>` with the name of your `OpenStackControlPlane` CR (for example, `openstack`) and `<namespace>` with the namespace where it is deployed (for example, `openstack`).
+* This patch does the following:
+** Sets `spec.keystone.template.customServiceConfig`. Ensure that you do not overwrite any previously defined value.
+** Defines `spec.keystone.template.extraMounts` to mount a secret named `keystone-domains` into the {identity_service} pods at `/etc/keystone/domains`. This secret contains your LDAP configuration files.
++
+[NOTE]
+You might need to wait a few minutes for the changes to propagate and for the {identity_service} pods to be updated.
```

Comment:
Do we have some way to verify if LDAP adoption succeeded? I see some tests in https://github.com/openstack-k8s-operators/data-plane-adoption/pull/970/files
To me it is worth to add Verification section, same we have for other services adoptions